packages/deploy/deploy/000_core/999_deploy_fake_polygon_operator_filter_registry.ts

```diff
@@ -0,0 +1,32 @@
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
```

Comment:
Why this naming for this deploy script? "999"

Reply:
Mock deploy script should go to `deploy/deploy_mocks`

Reply:
Good call, moved it to mocks and gave it sequential number.